### PR TITLE
feat: Update to v0.3!!!

### PR DESCRIPTION
**Pull request by EnokiUN**
don't ask about the labels

### CODE REVIEW

**Comment by ooliver1** [comment]
wontfix

File: docker-compose.yml [hunk 1]
```diff
@@ -1,40 +1,79 @@
...
+
+  mariadb:
+    <<: *restart-policy
+    image: mariadb:10-jammy
+    environment:
+      MYSQL_ROOT_PASSWORD: root
+      MYSQL_ROOT_HOST: "%"
+      MYSQL_DATABASE: eludris
+    volumes:
+      - "/maraidb:/var/lib/mysql"
```
Comment (ooliver1) [inline docker-compose.yml:79]: Isn't this linking to the host's root without a leading `.`?

**Comment by Chromosomologist** [comment]
good first issue tbh

File: docker-compose.yml (hunk 1 above)
Comment (EnokiUN) [reply docker-compose.yml:79]: yes, also maridb

**Comment by EnokiUN** [comment]
I wonder why

**Review by ooliver1 — APPROVED** [review]
It might work on a Monday.

**Comment by Chromosomologist** [comment]
very pog review I rate 8/8

**Review by Chromosomologist — APPROVED** [review]

**Pull request merged**